types/static-kdtree/tsconfig.json

```
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+      "esModuleInterop": true,
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

```suggestion
            "es6"
```
the JS module has no dependency on `dom` types, it uses stsandard language features (from lib).
This dependency is only added to satisfy tests dependency on `console` object. Can you replace/remove those and use e.g. type tests?:
```ts
idx; // $ExpectType number
```